core/src/main/scala/com/mesosphere/usi/core/Response.scala

```diff
@@ -0,0 +1,34 @@
+package com.mesosphere.usi.core
+
+import com.mesosphere.usi.models.{Mesos, StatusMessage}
+
+sealed trait Response {
+  def statusMessages: List[StatusMessage]
+
+  def mesosCalls: List[Mesos.Call]
+}
+
+case class ResponseWithMessages(statusMessages: List[StatusMessage] = Nil, mesosCalls: List[Mesos.Call] = Nil) extends Response
+
+case class ResponseBuilder(reverseStatusMessage: List[StatusMessage] = Nil, reverseMesosCalls: List[Mesos.Call] = Nil) extends Response {
```

Comment:
Maybe I misunderstand something but should the `ResponseBuilder` have a `build` method?

Reply:
It does not need one